[2020-04-26 20:24:48,411] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: ReclameAqui.persiste_dados_mongodb 2020-04-26T20:24:42.230047+00:00 [queued]>
[2020-04-26 20:24:48,485] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: ReclameAqui.persiste_dados_mongodb 2020-04-26T20:24:42.230047+00:00 [queued]>
[2020-04-26 20:24:48,486] {{taskinstance.py:866}} INFO - 
--------------------------------------------------------------------------------
[2020-04-26 20:24:48,487] {{taskinstance.py:867}} INFO - Starting attempt 1 of 2
[2020-04-26 20:24:48,488] {{taskinstance.py:868}} INFO - 
--------------------------------------------------------------------------------
[2020-04-26 20:24:48,550] {{taskinstance.py:887}} INFO - Executing <Task(PythonOperator): persiste_dados_mongodb> on 2020-04-26T20:24:42.230047+00:00
[2020-04-26 20:24:48,560] {{standard_task_runner.py:53}} INFO - Started process 6970 to run task
[2020-04-26 20:24:48,695] {{logging_mixin.py:112}} INFO - Running %s on host %s <TaskInstance: ReclameAqui.persiste_dados_mongodb 2020-04-26T20:24:42.230047+00:00 [running]> 030a07de9129.semantix
[2020-04-26 20:24:48,762] {{logging_mixin.py:112}} INFO - DEBUG POST: {'company': 'Cielo', 'date': '20190101'}
[2020-04-26 20:24:48,788] {{logging_mixin.py:112}} INFO - 200
[2020-04-26 20:24:48,879] {{logging_mixin.py:112}} INFO - {"id": "20190101CIELO"}
[2020-04-26 20:24:48,880] {{python_operator.py:114}} INFO - Done. Returned value was: None
[2020-04-26 20:24:48,918] {{taskinstance.py:1048}} INFO - Marking task as SUCCESS.dag_id=ReclameAqui, task_id=persiste_dados_mongodb, execution_date=20200426T202442, start_date=20200426T202448, end_date=20200426T202448
[2020-04-26 20:24:58,312] {{logging_mixin.py:112}} INFO - [2020-04-26 20:24:58,312] {{local_task_job.py:103}} INFO - Task exited with return code 0
